templates/users/employee/profile.html

```diff
@@ -23,4 +23,27 @@
             </div>
         </form>
     </div>
+    <script>
+    function changeThumbnail(){
```

Comment:
Why is this

Reply:
Prije uploda slike imas preview slike u avataru profila